cm/cm_heat_demand/tools/response.py

```diff
@@ -0,0 +1,101 @@
+import numpy as np
+
+
+def get_response(
+    total_potential: float, total_heat_demand: float, areas_potential: np.ndarray
+) -> dict:
+    """
+    Generate the the dictionary return by the CM.
+
+    Inputs :
+        * total_potential : total potential of all areas.
+        * total_heat_demand : total demand of all areas.
+        * areas_potential : potential of each area.
+
+    Output :
+        * response : dictionary that will be sent by the CM.
+    """
+
+    def get_graphs(
+        base_dictionary: dict,
+        areas_potential: np.ndarray,
+    ) -> dict:
+        """
+        Add information about the graphics to the based dictionary.
+
+        Inputs :
+            * base_dictionary : dictionary that will be updated.
+            * areas_potential : potential of each area.
+        Output :
+            * base_dictionary : updated dictionary.
+        """
+
+        base_dictionary["graphs"] = dict()
+
+        # Areas potential
+        if areas_potential[1:].size > 0:
+            print("areas_potential")
+            print(areas_potential[1:])
+            base_dictionary["graphs"]["Areas potentials (GWh)"] = dict()
+            base_dictionary["graphs"]["Areas potentials (GWh)"]["type"] = "bar"
+
+            # labels start from 1, therefore the array size is 'num_labels_array + 1'
+            generator = [element for element in areas_potential[1:] if element > 0]
+
+            values = [
+                ("Zone " + str(index + 1), value)
+                for index, value in enumerate(generator)
+            ]
```

Comment:
I don't understand the comment. What are you doing here?

Reply:
I browse the list `areas_potential` containing the potential of each area by checking that the potential is positive 
Then I generate a tuple, see below
```
generator = [10, 15, ...]
values = [("Zone 1", 10), ("Zone 2", 15), ...]
```
The `values` is return to the frontend

Reply:
But I don't understand the comment about the array size?

Reply:
I answered in this comment: https://github.com/enermaps/enermaps/pull/256#issuecomment-908196761